recipes/traitlets/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "4.2.1" %}
+
+package:
+  name: traitlets
+  version: {{ version }}
+
+source:
+  fn: traitlets-{{ version }}.tar.gz
+  url: https://github.com/ipython/traitlets/archive/{{ version }}.tar.gz
+  md5: fc7f46a76b99ebc5068f99033d268dcf
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - ipython_genutils
+    - decorator
+    # enum34 may be of interest here also, but a test would also
+    # need to be added to the recipe. 
```

Comment:
I guess I would be more nervous if it weren't a simple Python 3.4 backport. Though you make a good point that it probably should be tested especially since it is so easy to do.

Based on what I saw from the CI times of `mistune` and `traitlets`, this ends up being ~1min (with a few second hedge factor).

That being said, I don't think we should be so aggressive on avoiding running the test suites. Particularly when it comes to compiled software. This case isn't so bad as they are both pure Python. However, when it is something fairly low level that contains C/C++, we should want to avoid releasing it as potentially broken software if we can. This is as much to make sure the code is ok as it is to make sure the compiler is behaving correctly (this will be more important if we start shipping our own). For instance, take this quote from GMP.

> Most problems with GMP these days are due to problems not in GMP, but with the compiler used for compiling the GMP sources. This is a major concern to the GMP project, since an incorrect computation is an incorrect computation, whether caused by a GMP bug or a compiler bug. We fight this by making the GMP testsuite have great coverage, so that it should catch every possible miscompilation.

-- Reporting bugs in GMP - ( https://gmplib.org/ )

The case with most of the compiled code that has entered our stack to date fits under this category. It is normally quite old, very stable, and is widely used (both in the world and in our stack).

Especially given our recent improvements with CI performance, we shouldn't let fear of testing time drive poor release quality.